### ISSUE TITLE
Option to use chat_id in telegram bot notifier

### ISSUE THREAD

**Issue by terrytw** [Flexget/Flexget#4055]
### Expected behaviour:

```
my-task:
  notify:
    entries:
      message: Notification body.
      via:
        - telegram:
            bot_token: token
            parse_mode: markdown
            disable_previews: no
            recipients:
              - username: my-user-name
              - group: my-group-name
              - fullname:
                  first: my-first-name
                  sur: my-sur-name
              - chat_id:
```
It would be much more convenient if users can directly use chat_id to define telegram notification channel as shown above.

The problem with username and fullname is that you have to make sure there are recent messages in the chat, otherwise chat_id retrieval will fail. Using chat_id is simply more reliable. 

### Actual behaviour:
Now only username, fullname or group is allowed as config for the telegram notifier. 

### Steps to reproduce:
N/A

#### Config:
N/A
  
#### Log:
N/A

### Additional information:

- FlexGet version: 3.11.46
- Python version: 3.11
- Installation method: docker
- Using daemon (yes/no): yes
- OS and version: Synology 7.2
- Link to crash log: N/A

**Comment by vivodi** [comment]
Fixed in #4083, this can be closed now,

But one thing to note is that if the chat id you directly fill in has been migrated, it will cause an additional HTTP request each time the program runs. If your group has been migrated, you should fill in the new chat id.

If you are capable, you can also improve the relevant code (Store the old and new chat ids in the data table and make relevant judgments when the program runs.).

**Comment by terrytw** [comment]
@vivodi thank you so much for taking time to fix it!
